packages/components/src/components/list-box/_list-box.scss

```diff
@@ -105,7 +115,6 @@ $list-box-menu-width: rem(300px);
   // invalid states
   .#{$prefix}--list-box__invalid-icon {
     position: absolute;
-    top: $carbon--spacing-04;
```

Comment:
![image](https://user-images.githubusercontent.com/8265238/69408180-23f68e80-0cbb-11ea-96ab-f9f3f5f2fff6.png)

rather than removing the `top` rule, maybe we can go with

```scss
top: 50%;
transform: translateY(-50%);
```

Reply:
Thanks @emyarod for the suggestion - Fixed.